calibrators/mapping_based_calibrator/README.md

```diff
@@ -0,0 +1,238 @@
+| `base_lidar_crop_box_max_x`              | `double` | `20.0`        | Maximum x-coordinate in meters for the cropping box in base-lidar calibration to focus on relevant data areas.  |
+| `base_lidar_crop_box_max_y`              | `double` | `20.0`        | Maximum y-coordinate in meters for the cropping box in base-lidar calibration to focus on relevant data areas.  |
+| `base_lidar_crop_box_max_z`              | `double` | `20.0`        | Maximum z-coordinate in meters for the cropping box in base-lidar calibration to focus on relevant data areas.  |
+| `base_lidar_max_inlier_distance`         | `double` | `0.01`        | Maximum inlier distance in meters for ground extraction by using `pcl::SACSegmentation`.                        |
+| `base_lidar_max_iterations`              | `int`    | `1000`        | The maximum number of iterations for ground extraction by using `pcl::SACSegmentation`.                         |
+| `base_lidar_min_plane_points`            | `int`    | `1000`        | The minimum number of points required in a pointcloud, ensuring sufficient data for applying ground extraction. |
+| `base_lidar_min_plane_points_percentage` | `double` | `10.0`        | The minimum percentage of the ground plane points in a pointcloud.                                              |
+| `base_lidar_max_cos_distance`            | `double` | `0.5`         | Maximum cosine distance for applying ground plane extraction.                                                   |
+| `base_lidar_overwrite_xy_yaw`            | `bool`   | `false`       | Flag to allow overwriting the x, y, and yaw value during base-lidar calibration.                                |
+
+## Known issues/limitations
+
+- As described in [Step 2](#step-2-prepare-calibration-data), the calibrator interpolates the pose of the `mapping lidar` at the timestamp of the `calibration lidars`. Therefore, the calibrator highly requires time synchronization between all of the lidar sensors.
+- It is required that the `mapping lidar` have high resolution. For the `calibration lidar`, both high and low resolutions are acceptable. Therefore, the vehicle with only low-resolution lidars could not use the calibrator.
+- A good initial calibration parameters is required for the calibrator. Better initial calibration parameters could help the registration algorithm estimate a more reliable transformation.
+
+## Pro tips/recommendations
+
+- To build the map accurately, drive your vehicle at the lowest feasible speed, for example, 2 km/h would be a good speed. If the user drives the vehicle too fast, the pointcloud will be distorted.
+- To create an accurate map, the surroundings of the calibration area are crucial. Ensure that the environment is rich in natural landmarks suitable for registration-based mapping in all directions, which is shown in the image below. This will help the lidar capture sufficient details beyond simple features like lane surfaces or walls.
```

Comment:
Required features only for mapping?
What about calibration itself. Give several good examples of positive and negative examples for calibration